src/systemActs/ContentActs.ts

```diff
@@ -69,11 +70,17 @@ export class UnusableInputValueAct<T> extends ContentAct {
 
     render(input: ControlInput, controlResponseBuilder: ControlResponseBuilder): void {
         if (this.payload.renderedReason !== undefined) {
-            controlResponseBuilder.addPromptFragment(`Sorry, ${this.payload.renderedReason}.`);
+            controlResponseBuilder.addPromptFragment(
+                i18next.t('UNUSABLE_INPUT_VALUE_ACT_DEFAULT_PROMPT', {
+                    value: this.payload.renderedReason,
+                }),
+            );
         } else {
             throw new Error(
-                `Cannot directly render UnusableInputAct as payload.renderedReason is undefined. ${this.toString()}. ` +
-                    `Either provide a renderedReason when creating the act, or render the act in control.render() or controlManager.render()`,
+                i18next.t('CONTENT_ACT_ERROR_DEFAULT_PROMPT', {
```

Comment:
Errors are not usually localized. OK for these to be literals in en-US

Reply:
Ok, reverted back 👍